src/Utilities/InfoStream.h

```diff
@@ -0,0 +1,95 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2017 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//
+// File created by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+/** @file InfoStream.h
+ * @brief Declaration of InfoStream class.
+ */
+
+#ifndef INFOSTREAM_H
+#define INFOSTREAM_H
+
+#include <string>
+#include <iostream>
+#include <iomanip>
+#include <sstream>
+
+/**
+ *  Interface to output streams.  Can redirect output to stdout/stderr, a file, or a null stream.
+ */
+
+class InfoStream
+{
+public:
+  InfoStream(std::ostream *output_stream): prevStream(NULL), nullStream(new std::ostream(NULL)),
```

Comment:
nullStream was not freed in the destructor. This is one of the leak I mentioned this morning. Please fix.